projects/event-api.md

```diff
@@ -0,0 +1,90 @@
+### Description
+
+The Client / RUM Sig is looking to finalize, complete and resolve the outstanding issues / PR's that exist around the Event API / SDK and the Semantic Conventions that defines what an "event" is.
```

Comment:
I see there's a project board [here](https://github.com/orgs/open-telemetry/projects/65). Is this description meant to imply that the working group should try to resolve all of these issues?

Reply:
No, it will be the responsibility of the of the Event WG to determine which of the "related" issues should / need to be addressed and therefore agree about which identified issues (in all categories) the group will address.

I took a first cut of going through all of the issues to identify not just what has been planned during the Client RUM Sig but also other issues that have existed for some time that appear to be related to the goals / deliverables of this group. Some of the identified issues appeared to be related enough as to provide some meaningful feedback on what people are looking for as well as what should be considered.